File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False
        temp = x
        rev = 0
        while temp >= 10:
            mod = temp%10
            rev = (rev *10)+mod
            temp //=10
        rev = (rev *10)+temp
        print(rev)
        return rev == x

File: 9-palindrome-number/palindrome-number.go
func isPalindrome(x int) bool {
    var b int
    var t int = x
    if  x < 0 {
        return false
    }
    for x > 0{
        b = b * 10 + x % 10
        x = x / 10
    }
    if t == b {
        return true
    }else {
        return false
    }
    
}